tinytga/Cargo.toml

```
@@ -18,6 +18,9 @@ exclude = [
 [badges]
 circle-ci = { repository = "jamwaffles/embedded-graphics", branch = "master" }
 
+[dependencies]
+paste = "1.0"
```

Looks like this is only used in the tests, so should be a dev-dependency I think.

```suggestion
[dev-dependencies]
paste = "1.0"
```